articles/quickstart/native/react-native-expo/00-login.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to an Expo application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - react-native
+  - expo
+github:
+  path: 00-Login-Expo
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD012 MD041 -->
+::: warning
+This SDK is not compatible with "Expo Go" app. It is compatible only with Custom Dev Client and EAS builds
+:::
+
+<%= include('../_includes/_getting_started', { library: 'Expo'}) %>
+
+## Install Dependencies
+
+How to install the React Native Auth0 module.
+
+::: note
+Please refer to the [official documentation](https://docs.expo.dev/) for additional details on Expo.
+:::
+
+### Yarn
+
+```bash
+yarn add react-native-auth0
+```
+
+::: note
+For further reference on yarn, check [their official documentation](https://yarnpkg.com/en/docs).
+:::
+
+### npm
+
+```bash
+npm install react-native-auth0 --save
+```
+
+The first step in adding authentication to your application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the hosted [login page](/hosted-pages/login).
+
+<div class="phone-mockup"><img src="/media/articles/native-platforms/ios-swift/login-ios.png" alt="Universal Login"></div>
+
+## Integrate Auth0 in Your Application
+
+### Setup Auth0 Config Plugin
+
+The Auth0 package runs custom native code that needs to be configured at build time. We will use [Expo Config Plugin](https://docs.expo.dev/guides/config-plugins/) to achieve this.
+
+The `react-native-auth0` plugin will be added in the [Expo Config](https://docs.expo.dev/workflow/configuration/) file at `app.json` or `app.config.js`.
+
+```json
+{
+  "expo": {
+    ...
+    "plugins": [
+      [
+        "react-native-auth0",
+        {
+          "domain": "{DOMAIN}"
```

Comment:
We should use the correct placeholder here so that the docs app is able to replace it with the domain of the tenant that the reader is currently logged in with:

```suggestion
          "domain": "${account.namespace}"
```